packages/react-components/react-charts-preview/src/components/LineChart/useLineChartStyles.styles.ts

```diff
@@ -0,0 +1,56 @@
+import { makeStyles, mergeClasses, shorthands } from '@griffel/react';
+import { tokens } from '@fluentui/react-theme';
+import { ILineChartProps, ILineChartStyles } from './LineChart.types';
+import { SlotClassNames } from '@fluentui/react-utilities/src/index';
+
+/**
+ * @internal
+ */
+export const linechartClassNames: SlotClassNames<ILineChartStyles> = {
+  tooltip: 'fui-lc__tooltip',
+  root: '',
+  xAxis: '',
+  yAxis: '',
+  legendContainer: '',
+  hover: '',
+  calloutContentRoot: '',
+  calloutContentX: '',
+  calloutContentY: '',
+  descriptionMessage: '',
+  calloutDateTimeContainer: '',
+  calloutInfoContainer: '',
+  calloutBlockContainer: '',
+  calloutlegendText: '',
+  axisTitle: '',
+  chartTitle: '',
+  opacityChangeOnHover: '',
+  shapeStyles: '',
+  chartWrapper: '',
+};
+
+/**
+ * Base Styles
+ */
+const useStyles = makeStyles({
+  tooltip: {
+    display: 'flex',
+    flexDirection: 'column',
+    ...shorthands.padding('8px'),
+    position: 'absolute',
+    textAlign: 'center',
+    top: '0px',
+    fill: tokens.colorNeutralBackground1,
```

Comment:
I see background property is missing

Reply:
Okay, will add it.

Reply:
Won't the fill property do work for background ?